.travis.yml

```diff
@@ -40,3 +48,31 @@ script:
   - curl -k https://localhost:444
   - curl -k https://localhost:8443
   - curl -k https://localhost:8444
+
+after_success:
+  - curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+  - chmod +x ./kubectl
+  - sudo mv ./kubectl /usr/local/bin/kubectl
+  - echo -n ${travis} | base64 -d > travis-service-account.json
+  - gcloud auth activate-service-account travis@api-platform-demo-209807.iam.gserviceaccount.com --key-file travis-service-account.json --project=api-platform-demo-209807
```

Comment:
`travis@api-platform-demo-209807.iam.gserviceaccount.com` should be stored in a Travis env var (secret), same for the project name and the zone.